app/api/helpers/storage.py

```diff
@@ -149,7 +149,10 @@ def upload_local(uploaded_file, key, **kwargs):
     if not os.path.isdir(dir_path):
         os.makedirs(dir_path)
     uploaded_file.save(file_path)
-    return '/serve_' + file_relative_path
+    image_url = '/serve_' + file_relative_path
```

Comment:
Why the `/serve_` path ? 

Make it such that, 

If the file is inside the static folder, (like `<project root>/static/filename.png`), it should be accessible at `/static/filename.png` 

And if a stattic domain is set, url returned should be `<staticdomain>/filename.png`